packages/api/cms-api/src/dam/files/dto/file.body.ts

```diff
@@ -44,6 +47,15 @@ export function createUploadFileBody({ Scope }: { Scope: Type<DamScopeInterface>
         @ClassTransformerType(() => ImageCropAreaInput)
         @ValidateNested()
         imageCropArea?: ImageCropAreaInput;
+
+        @IsString()
+        @IsNotEmpty()
+        @ValidateIf((input) => input.importSourceType !== undefined)
+        importSourceId?: string;
+
+        @IsString()
+        @IsUndefinable()
+        importSourceType?: string;
```

Comment:
I just realised that we also need this here:

```suggestion
        @IsString()
        @IsNotEmpty()
        @ValidateIf((input) => input.importSourceId !== undefined)
        importSourceType?: string;
```